webpack.config.js

```diff
@@ -17,6 +17,14 @@ module.exports = {
             presets: ['@babel/preset-env', '@babel/preset-react']
           }
         }
+      },
+      {
+        test: /\.(png|jpe?g|gif)$/i,
+        use: [
+          {
+            loader: 'file-loader',
+          },
+        ],
```

Comment:
Looks like you might be missing a `}` here.

Reply:
I didn't think so, but I'll take a look.